doc/aspice_swe3_4/example/iceoryx_component/include/example_component/example_sample_class.hpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef MY_SAMPLE_CLASS_HPP_INCLUDED
+#define MY_SAMPLE_CLASS_HPP_INCLUDED
```

Comment:
Our include guard pattern is different:
- IOX_[COMPONENT_W/O_ICEORYX]_[SUB_COMPONENT]_[FILE_NAME]
- IOX_[COMPONENT_W/O_ICEORYX]_[SUB_COMPONENT]_[SUB_COMPONENT]_[FILE_NAME]

In this case it'd look like this: IOX_COMPONENT_EXAMPLE_COMPONENT_EXAMPLE_SAMPLE_CLASS_HPP

Reply:
i reworked the include guards and added the schema to the documenentation.